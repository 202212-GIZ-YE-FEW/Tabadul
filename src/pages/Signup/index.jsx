import { serverSideTranslations } from "next-i18next/serverSideTranslations";
import React from "react";

import Footer from "@/components/Footer/Footer";
import Navbar from "@/components/Navbar/Navbar";
import Signup from "@/components/Sign UP/Index";

function SignUp() {
    return (
        <div>
            <Navbar />
            <Signup />
            <Footer />
        </div>
    );
}

export default SignUp;

export async function getStaticProps({ locale }) {
    return {
        props: {
            ...(await serverSideTranslations(locale, ["signup"])),
            // Will be passed to the page component as props
        },
    };
}
