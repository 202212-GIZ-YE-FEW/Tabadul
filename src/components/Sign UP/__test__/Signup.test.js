import renderer from "react-test-renderer";

import Signup from "../Index";

it("renders correctly", () => {
    const tree = renderer.create(<Signup />).toJSON();
    expect(tree).toMatchSnapshot();
});
