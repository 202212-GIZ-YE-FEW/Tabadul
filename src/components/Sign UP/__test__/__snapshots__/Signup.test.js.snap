// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-0"
>
  <h2
    className="css-143k8nc"
  >
    SignUp
  </h2>
  <div
    className="css-1ieito0"
  >
    <form
      className="css-0"
      onSubmit={[Function]}
    >
      <div
        className="css-vaakcf"
      >
        <label
          className="css-18drj25"
        >
          SignUpName
        </label>
        <input
          className="css-1uo441q"
          id="name"
          name="name"
          onChange={[Function]}
          placeholder="EnterYourname"
          type="text"
          value=""
        />
      </div>
      <div
        className="css-vaakcf"
      >
        <label
          className="css-18drj25"
        >
          SignupEmail
        </label>
        <input
          className="css-1uo441q"
          id="email"
          name="email"
          onChange={[Function]}
          placeholder="EnterYouremail"
          type="email"
          value=""
        />
      </div>
      <div
        className="css-vaakcf"
      >
        <label
          className="css-18drj25"
        >
          Password
        </label>
        <input
          autoComplete="current-password"
          className="css-1uo441q"
          id="password"
          name="passwprd"
          onChange={[Function]}
          placeholder="Enteryourpassword"
          type="password"
          value=""
        />
      </div>
      <div
        className="css-vaakcf"
      >
        <label
          className="css-18drj25"
        >
          ConfirmPassword
        </label>
        <input
          className="css-1uo441q"
          id="confirmpassword"
          name="confirmpassword"
          onChange={[Function]}
          placeholder="Confirmyourpassword"
          type="password"
          value=""
        />
      </div>
      <div
        className="css-vaakcf"
      >
        <label
          className="css-18drj25"
        >
          Phone
        </label>
        <input
          className="css-1uo441q"
          id="phone"
          name="phone"
          onChange={[Function]}
          placeholder="EnteryourPhone"
          type="number"
          value=""
        />
      </div>
      <div
        className="css-vaakcf"
      >
        <label
          className="css-18drj25"
        >
          Location
        </label>
        <select
          className="css-kgetvl"
          id="location"
          name="location"
          onChange={[Function]}
        >
          <option
            className="css-0"
            disabled={true}
            selected={true}
          >
            SelectLocation
          </option>
          <option
            className="css-0"
          >
            Mukalla
          </option>
          <option
            className="css-0"
          >
            Istanbul
          </option>
          <option
            className="css-0"
          >
            Sana'a
          </option>
          <option
            className="css-0"
          >
            egypt
          </option>
        </select>
      </div>
      <div
        className="css-omk73n"
      >
        <button
          className="css-1ittbjx"
          type="submit"
        >
          <span
            className="css-f4tyiu"
          >
            SignUPbutton
          </span>
        </button>
        <p
          className="css-1057m3k"
        >
          SignUpwith
        </p>
        <div
          className="css-1mzzuk6"
        >
          <img
            alt="Facebook"
            className="css-1vai4q9"
            data-nimg="1"
            decoding="async"
            height={42.3}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/images/Facebook.svg"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={42.3}
          />
          <img
            alt="Instagram"
            className="css-1vai4q9"
            data-nimg="1"
            decoding="async"
            height={42.3}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/images/Instagram.svg"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={42.3}
          />
          <img
            alt="Linkedin"
            className="css-1vai4q9"
            data-nimg="1"
            decoding="async"
            height={42.3}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/images/Linkedin.svg"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={42.3}
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;
